android/build.gradle

```diff
@@ -39,6 +42,10 @@ apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 
 android {
+    if (getGradle().getGradleVersion().substring(0, 1).toInteger() >= 8) {
+        namespace "com.doordeck_flutter_plugin"
+    }
```

Comment:
Can we remove this and instead use add the Gradle wrapper so we have control over the Gradle version.

Reply:
Hmm I see the gradle wrapper is kind of here but it's a bit broken... re-run `./gradlew wrapper` if you can!

Reply:
I've seen this as a `namespace` incompatibility with the users using Flutter with gradle `< 8`
https://stackoverflow.com/a/76174983

Reply:
I fully agree there could be backwards compatible issues, but we should only support running Gradle through the wrapper so we have control over the version and won't run into that issue; the wrapper setup just seems a bit broken, the `gradlew` file is missing (amongst others) but the `gradle/wrapper/gradle-wrapper.properties` file is there!

Reply:
This is what happens.

If a Flutter App uses a wrapper >= 8 and the gradle dependency classpath >= 8, there's an error
![image](https://github.com/doordeck/doordeck-sdk-flutter/assets/6842786/d7724ab4-a428-45a3-9be7-b6262a5a8e37)

if I don't add the conditional
```
if (getGradle().getGradleVersion().substring(0, 1).toInteger() >= 8) {
```

and we're below 8, I've seen cases of warning but no error. So I've left the namespace in it.
Would that be enough?